### PR TITLE
/// MEGA UPDATE ///

### PR DESCRIPTION
**Pull request by Stev3eM**
/// MEGA UPDATE ///

-> created _ProfilePage.tsx_ and implemented ~70% of mockup design
-> created _ProfilePage.css_ 
-> removed global background color grey-blue (#38527D) in _Frontend.css_ -> it was impossible to set background color in other pages to transparent
-> set the background color of all pages in their related CSS-pages back to grey-blue (#38527D)
-> refactored _FAQsPage.tsx_ into _/components_ directory
-> created _ImpressumPage.css_
-> created _RecipePage.css_
-> created _RecipeBrowser.css_
-> created _RecipeCreator.css_
-> created _SettingsPage.tsx_
-> created _SettingsPage.css_
-> created _AboutUsPage.css_
-> Added _route path_ to all new component pages in App.tsx
-> refactored _Frontend.css_ into _LandingPage.css_
-> created _Global.css_ to contain all global CSS settings
-> added several component images into assets directory

/// KNOWN BUGS ///

-> removing global background color and replacing it with transparent background color caused to some pages containg now a white edge ([Issue #75](https://github.com/SE-TINF22B6/CookHub/issues/75#issue-2019665679) has been created)

---------------------------------------------------------------------------------------------------------------------------------

/// UPDATE ///

-> I have merged **MEGA UPDATE 2** (https://github.com/SE-TINF22B6/CookHub/pull/76) from _RecipeCreator-implementation branch_ into **MEGA UPDATE** _frontend-profilpage-imp branche_ (https://github.com/SE-TINF22B6/CookHub/issues/75) and checked functionality.

What makes this enriched merge request to  **/// DOUBLE MEGA UPDATE XXL ///**  !!!

_-> still problems with background colors_

### CODE REVIEW

File: Frontend/src/assets/img_placeholder_icon.webp
Comment (Challenger3010) [inline Frontend/src/assets/img_placeholder_icon.webp:1]: do we need a placerholder as .webp?

File: Project_Doc/Diagrams/uml-class-diagram.drawio.svg
Comment (Challenger3010) [inline Project_Doc/Diagrams/uml-class-diagram.drawio.svg:1]: why did you rename the profilpage.tsx to desktop1.tsx?

**Review by Challenger3010 — COMMENTED** [review]
Please check the comments

File: Frontend/src/assets/img_placeholder_icon.webp
Comment (Stev3eM) [reply Frontend/src/assets/img_placeholder_icon.webp:1]: I was just lazy. Can be replaced with user avatar and user name in the future.

File: Project_Doc/Diagrams/uml-class-diagram.drawio.svg
Comment (Stev3eM) [reply Project_Doc/Diagrams/uml-class-diagram.drawio.svg:1]: Where?

Comment (Challenger3010) [reply Project_Doc/Diagrams/uml-class-diagram.drawio.svg:1]: UserClient.ts at the bottom there are 3 relations and you renamed one
